torch/_dynamo/optimizations/distributed.py

```diff
@@ -88,10 +100,16 @@ def compile_fn(self, gm: fx.GraphModule, example_inputs: List[torch.Tensor]):
             if node.op == "call_module":
                 target = gm.get_submodule(node.target)
                 for name, p in target.named_parameters():
+                    if (str(node), name) in self.parameters_to_ignore:
```

Comment:
mirroring chat thread, @zdevito and @voznesenskym recommended `node.target in self.parameters_to_ignore` to avoid relying on fx naming behavior

Reply:
oh i see . def we should do this then.

Reply:
yea, i wish. node.target is `self_net_2` and name is `weight`
DDP tells us to ignore "net.2.weight".

I agree that this fx_name thing is unappealing.